conda/cli/conda_argparse.py

```diff
@@ -958,6 +952,47 @@ def configure_parser_remove(sub_parsers, name='remove'):
     p.set_defaults(func='.main_remove.execute')
 
 
+def configure_parser_run(sub_parsers):
+    help = "Run an executable in a conda environment. [Experimental]"
+    descr = help + dedent("""
+
+    Use '--' (double dash) to separate CLI flags for 'conda run' from CLI flags sent to
+    the process being launched.
+
+    Example usage:
+
+        $ conda create -y -n my-python-2-env python=2
+        $ conda run -n my-python-2-env python -- --version
+    """)
+
+    epilog = dedent("""
+    """)
+
+    p = sub_parsers.add_parser(
+        'run',
+        description=descr,
+        help=help,
+        epilog=epilog,
+    )
+
+    add_parser_prefix(p)
+    p.add_argument(
+        "-v", "--verbose",
+        action=NullCountAction,
+        help="Use once for info, twice for debug, three times for trace.",
+        dest="verbosity",
+        default=NULL,
+    )
+
+    p.add_argument(
+        'executable_call',
+        nargs=ONE_OR_MORE,
```

Comment:
You'll have to use `REMAINDER` here. Otherwise, e.g.,
```sh
conda run -v -p /p target -v
```
will be parsed as
```python
{'v': 2, 'p': '/p', 'executable_call': 'target'}
```